geoh5py/objects/object_base.py

```diff
@@ -214,47 +212,84 @@ def comments(self):
 
         return None
 
+    def copy(
+        self,
+        parent=None,
+        copy_children: bool = True,
+        clear_cache: bool = False,
+        mask: list[float] | np.ndarray | None = None,
```

Comment:
**question:** if the mask is a list of vertex indices, why does it contain float instead of integer?

Reply:
Should be `int`